__tests__/app.test.js

```diff
@@ -45,13 +45,40 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: sends an article object containing details of the required article", () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.length).toBe(1);
+      });
```

Comment:
While this test is fine, I think we could be more certain that we have got the expected article if we asserted against some of the properties and expected values